catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/AccessUtil.java

```diff
@@ -86,4 +87,39 @@ private AccessUtil() {
     }
     return serializables.stream().map(type::cast).collect(Collectors.toList());
   }
+
+  @SuppressWarnings("unchecked")
+  @Nullable
+  public static <T> List<T> safeGetList(Map map, String key, Class<T> type) {
+    List unchecked = safeGet(map, key, List.class);
+    if (unchecked == null) {
+      return null;
+    }
+    try {
+      return (List<T>) unchecked.stream().map(type::cast).collect(Collectors.toList());
+    } catch (ClassCastException e) {
```

Comment:
Should we be using exceptions to control this flow?

Reply:
Off the top of my head, I can definitely think of a better way to do it. But since this code is being tested outside of it's own test, I made a separate ticket to address this: 
https://codice.atlassian.net/browse/DDF-3834